File: programs/c-programming/array/maxMin.c
#include <stdio.h>

int* maxMin(int a[], int size) {
    int result[2];
    int maximum = -99999;
    int minimum = 99999;
    for (int i = 0; i < size; i++) {
        if (maximum < a[i]) {
            maximum = a[i];
        }
        if (minimum > a[i]) {
            minimum = a[i];
        }
    }
    result[0] = maximum;
    result[1] = minimum;
    return result;
}

int main() {
    int a[10] = {15, 10, 2, 49, 35, 30, 5, 17, 37, 9};
    int *result = maxMin(a, 10);
    printf("Maximum: %d\n", result[0]);
    printf("Minimum: %d\n", result[1]);
    return 0;
}

File: programs/python-programming/array/max_min.py
def max_min(nums):
    maximum = -99999
    minimum = 99999
    for num in nums:
        if num > maximum:
            maximum = num
        if num < minimum:
            minimum = num
    return maximum, minimum

if __name__ == '__main__':
    arr = [15, 10, 2, 49, 35, 30, 5, 17, 37, 9]
    maximum, minimum = max_min(arr)
    print(maximum)
    print(minimum)